packs/data/equipment.db/clockwork-macuahuitl.json

```
@@ -22,7 +22,7 @@
             "value": ""
         },
         "description": {
-            "value": ""
+            "value": "<p>This finely-made wooden club has a beautiful, lacquered finish that gleams in the sunlight. A heavy ring of gears lined with sharpened pieces of obsidian automatically and constantly spin around the bulk of the wooden club lengthwise. Striking a foe digs the obsidian gears into the enemy's flesh and tears it with its blades. The ever-turning gears also help to dislodge an enemy's defensive position against the weapon.</p>\n<p>The clockwork macuahuitl deals 1d10 slashing damage and has the backswing, forceful, and versatile B traits. The clockwork macuahuitl is a two-handed advanced weapon in the club weapon group.</p>"
```

```suggestion
            "value": "<p>This finely-made wooden club has a beautiful, lacquered finish that gleams in the sunlight. A heavy ring of gears lined with sharpened pieces of obsidian automatically and constantly spin around the bulk of the wooden club lengthwise. Striking a foe digs the obsidian gears into the enemy's flesh and tears it with its blades. The ever-turning gears also help to dislodge an enemy's defensive position against the weapon.</p>"
```
We only include the actual description.